### PR TITLE
roll to version 0.5.4

### PR DESCRIPTION
**Pull request by caryan**

### CODE REVIEW

**Comment by hhromic** [comment]
`msgpack` is now in version `0.5.6`.
Also, the name/repository changed from `msgpack-python` to simply `msgpack` in PyPi [1].
Can this recipe be updated accordingly? Thanks!

[1] https://pypi.python.org/pypi/msgpack

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by jakirkham** [comment]
Could you please raise those in two separate issues, @hhromic? Thanks

**Comment by hhromic** [comment]
Sure no problem.
Also, what is the difference between this package and the one in conda defaults?
Wouldn't it be better to update the one in defaults and eliminate duplicate efforts?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {% set name = "msgpack-python" %}
-{% set version = "0.5.1" %}
+{% set version = "0.5.4" %}
 {% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "69aa1eb0e13be1d3bd495ca937eae66df4431126f5cfd5491dc40370e5644853" %}
+{% set hash_type = "md5" %}
+{% set hash = "6abd8cca9caa61c7dcb9b9214f9e600f" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Please go to [pypi.org]( https://pypi.org/ ) and grab the `sha256` checksum from there. `md5` is too broken to be usable.

**Comment by jakirkham** [comment]
`defaults` is essentially mirroring our packages. They aren't totally the same, but they do use our recipes and we make an effort to keep them in sync.

File: recipe/meta.yaml (hunk 1 above)
Comment (hhromic) [reply recipe/meta.yaml]: @caryan , can you update from 0.5.4 to 0.5.6 aswell ?  thanks!

**Comment by hhromic** [comment]
aah great to know.  I just created one extra issue for the name change as this PR should fix the version upgrade.

File: recipe/meta.yaml (hunk 1 above)
Comment (caryan) [reply recipe/meta.yaml]: Done. Sorry about the md5 change.  It took me a while to figure out that `pypi.org` is the new version of `pypi.python.org` and has the `sha256` hashes.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {% set name = "msgpack-python" %}
-{% set version = "0.5.1" %}
+{% set version = "0.5.6" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: So the latest version under the name `msgpack-python` is 0.5.5. Hence the CIs are failing to download 0.5.6. For now, let's do 0.5.5 and we can figure out what to do after once issue ( https://github.com/conda-forge/msgpack-python-feedstock/issues/5 ) is addressed.

Comment (caryan) [reply recipe/meta.yaml]: Dope! I'll get there eventually. Thanks for being so patient.

**Comment by jakirkham** [comment]
@pmlandwehr, could you please take a look?

**Comment by pmlandwehr** [comment]
Looks good!

**Pull request merged**